CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ chandler is in a pre-1.0 state. This means that its APIs and behavior are subjec
 
 ## [Unreleased][]
 
-* Your contribution here!
```

Comment:
Could you re-add the "Your contribution here!" line? I like to keep it there as a signpost for future contributors.